index.js

```diff
@@ -150,7 +150,7 @@ module.exports = function() {
         {
           label: 'Quit',
           accelerator: 'Command+Q',
-          click: function() { app.quit(); }
```

Comment:
Hi, I tried it and it works, but you're probably right that it's safer to keep it in a function scope.